src/index.js

```diff
@@ -17,6 +17,42 @@ class MyObjects {
   }
 }
 
+// remove todo function
+const removeTodo = (todo) => {
+  todoMainContainer.removeChild(todo);
+  // let count = 0;
+  const DataFromLocalStorage = JSON.parse(localStorage.getItem('list'));
+  function complete(i) { i.completed = false; }
+  const data = Array.from(DataFromLocalStorage).filter(complete);
+  // function counts(i) { i.index = count += 1; }
+  // data.map(counts);
+  localStorage.setItem(('list'), JSON.stringify(data));
+};
```

Comment:
**[REQUIRED]** As of the project, the requirement makes sure to create a new javascript file for the new functionalities since your `index.js` file is very big it's not readable anymore. 

**[REQUIRED]** Also, make sure to save your to-do's to `local storage` and it should persist when the user reloads the page. 

https://user-images.githubusercontent.com/34578631/172945725-92638c77-c220-4ba9-b7e2-af3dfbfc0ecd.mov